1.0.2-draft/README.md

```diff
@@ -0,0 +1,91 @@
+
+## 1. Purpose
+
+This specification attempts to create a standard for encoding of tiled geospatial data that can be shared across clients.
+
+## 2. File format
+
+The vector tile encoding scheme encodes vector data for a tile in a space efficient manner. It is designed to be used in browsers or serverside applications for fast rendering or lookups of feature data.
+
+Vector Tiles use [Google Protocol buffers](https://developers.google.com/protocol-buffers/) as a container format. It is exclusively geared towards square pixel tiles in [Spherical Mercator projection](http://wiki.openstreetmap.org/wiki/Mercator).
+
+## 3. Internal structure
+
+A vector tile SHOULD consist of one or more named layers. 
+
+A layer MUST contain a field describing its name and a version of the vector tile spec from which it was created. A layer  MUST also contain one or more features. A layer MAY contain one or more sets of feature attributes.
+
+Features MAY contain an `id` and `tags` (attributes) and MUST contain either a `geometry` (either point, linestring, or polygon) or a `raster` field. A feature SHALL NOT contain both a `raster` and a `geometry` field. If a feature has a `geometry` field it MUST also have a `type` field to describe the `geometry`. 
+
+### 3.1. Geometry Encoding
+
+Geometries are stored as an a single array of integers that represent a command,x,y stream (where command is a rendering command like `move_to` or `line_to`). Commands are encoded only when they change.
+
+Geometries with multiple parts (multipoint, multiline, or multipolygon) MUST be encoded one after another in the same `geometry` field and therefore are "flattened". Geometries with only a single part MUST have only a single `move_to` present. For multipoints and multilines a repeated `move_to` SHALL indicate another part of a multipart geometry. For polygons a repeated `move_to` SHALL indicate either another exterior of a new polygon part or an interior ring of the previous polygon part. The winding order MUST be used to distinguish between the two types: polygon interior rings (holes) must be oriented with the opposite winding order than their parent exterior rings and all interior rings MUST directly follow the exterior ring to which they belong. 
+
+The exterior ring MUST have a positive area as calculated by applying the [surveyor's formula](https://en.wikipedia.org/wiki/Shoelace_formula) to the vertices of the polygon in tile coordinates. In screen coordinates (with the Y axis positive down) this makes the exterior ring's winding order appear clockwise. In a frame where the Y axis is positive up, this would make the winding order appear counter clockwise.
+
+Geometry collections are not supported.
+
+Geometries SHOULD be clipped, reprojected into spherical mercator, converted to screen coordinates, and MUST be [delta](http://en.wikipedia.org/wiki/Delta_encoding) and [zigzag](https://developers.google.com/protocol-buffers/docs/encoding#types) encoded.
```

Comment:
> SHOULD be clipped, reprojected into spherical mercator

Geometries are stored as graphics in the tiles, as they are projected, clipped and converted to screen coordinates.
The client doesn't have to know about what projection the geometries were before that process, as it consumes only the final screen coordinates.
I wouldn't word this as SHOULD IMO